File: server/plugin/pcdn/router/bill.go
package router

import "github.com/gin-gonic/gin"

// BillRouter 账单路由（admin 组）
type BillRouter struct{}

// InitBillRouter 挂载账单路由
func (r *BillRouter) InitBillRouter(group *gin.RouterGroup) {
	group.POST("bill/generate", billApi.GenerateBill)
	group.GET("bill/list", billApi.GetBillList)
	group.GET("bill/find", billApi.GetBill)
	group.PUT("bill/approve", billApi.ApproveBill)
	group.PUT("bill/reject", billApi.RejectBill)
	group.PUT("bill/pay", billApi.PayBill)
}

File: web/src/plugin/pcdn/api/bill.js
import service from '@/utils/request'

export const generateBill = (data) => service({ url: '/pcdn/admin/bill/generate', method: 'post', data })
export const getBillList = (params) => service({ url: '/pcdn/admin/bill/list', method: 'get', params })
export const getBill = (params) => service({ url: '/pcdn/admin/bill/find', method: 'get', params })
export const approveBill = (data) => service({ url: '/pcdn/admin/bill/approve', method: 'put', data })
export const rejectBill = (data) => service({ url: '/pcdn/admin/bill/reject', method: 'put', data })
export const payBill = (data) => service({ url: '/pcdn/admin/bill/pay', method: 'put', data })
